### PR TITLE
Not a submission

### PR DESCRIPTION
**Pull request by ghost**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/89175)

### CODE REVIEW

**Comment by MooseOnTheLoose** [comment]
> 

@microsoft-github-policy-service agree [company="{BAR Team}"]

**Comment by MooseOnTheLoose** [comment]
@microsoft-github-policy-service agree company="BAR Team"

File: manifests/b/BAR_Team/Beyond-All-Reason/1.1779.0/BAR_Team.Beyond-All-Reason.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: BAR_Team.Beyond-All-Reason
+PackageVersion: 1.1779.0
+PackageLocale: en-US
+Publisher: BAR Team
```
Comment (russellbanks) [inline manifests/b/BAR_Team/Beyond-All-Reason/1.1779.0/BAR_Team.Beyond-All-Reason.locale.en-US.yaml:7]:
```suggestion
Publisher: gajop
```

This must match control panel

File: manifests/b/BAR_Team/Beyond-All-Reason/1.1779.0/BAR_Team.Beyond-All-Reason.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: BAR_Team.Beyond-All-Reason
+PackageVersion: 1.1779.0
+UpgradeBehavior: install
+Installers:
```
Comment (russellbanks) [inline manifests/b/BAR_Team/Beyond-All-Reason/1.1779.0/BAR_Team.Beyond-All-Reason.installer.yaml:7]:
```suggestion
AppsAndFeaturesEntries:
- DisplayName: Beyond-All-Reason 1.1779.0
  ProductCode: a5671b31-085d-5fba-830a-66a891c6a4a4
Installers:
```

**Review by russellbanks — CHANGES_REQUESTED** [review]
firefox_zyO3TSTe3q

**Comment by russellbanks** [comment]
I'll test in a sandbox when I can. I also need to check about the Identifier, as the dashes and underscores are a little inconvenient but if they're necessary, a moniker would be suitable so users could install it with `winget install beyondallreason`, for example.

File: manifests/b/BAR_Team/Beyond-All-Reason/1.1779.0/BAR_Team.Beyond-All-Reason.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+Author: BAR Team
+PackageName: Beyond All Reason
+PackageUrl: https://github.com/beyond-all-reason/BYAR-Chobby/releases
+License: GNU General Public License v2.0
+LicenseUrl: https://raw.githubusercontent.com/beyond-all-reason/Beyond-All-Reason/master/LICENSE.md
+Copyright: Copyright © 2022 BAR Team
+CopyrightUrl: https://raw.githubusercontent.com/beyond-all-reason/Beyond-All-Reason/master/LICENSE.md
+ShortDescription: Real-Time Strategy Redefined
+Description: Free Real-Time Strategy Game with a grand scale and full physical simulation in a sci-fi setting.
+Moniker: BAR
```
Comment (russellbanks) [inline manifests/b/BAR_Team/Beyond-All-Reason/1.1779.0/BAR_Team.Beyond-All-Reason.locale.en-US.yaml:20]:
```suggestion
Moniker: bar
```
Monikers are meant to be lowercase. Either `bar` or `beyondallreason` would work here.

**Review by russellbanks — CHANGES_REQUESTED** [review]
Please fix the moniker to be lowercase. The only other thing is the PackageIdentifier. It does seem a little awkward to have `_` in it. Does the publisher have any indication of their preference? The Publisher ARP Entry is `gajop`, but I can see the referenced pull request to change this. If BAR Team is preferable, I would suggest `BAR-Team` or `BARTeam`, unless the Publisher has used or requested `BAR_Team` specifically.

**Comment by russellbanks** [comment]
@MooseOnTheLoose

**Comment by MooseOnTheLoose** [comment]
I will not be proceeding with this submission for the time being.

**Pull request closed without merging**